MotorControl/MotorControl/Resources/Kinetic_Tremor.json

```diff
@@ -0,0 +1,178 @@
+                            "icon":"ComfortablePlaceToSitRose",
+                            "title": "COMFORTABLE PLACE TO SIT"
+                          }
+                        ],
+                "actions":{
+                    "goForward":{
+                        "type":"default",
+                        "buttonTitle":"Get Started"
+                    },
+                    "skip":{
+                        "type":"reminder",
+                        "reminderIdentifier":"Kinetic Tremor"
+                    },
+                    "moreInformation": {
+                        "type": "videoView",
+                        "buttonTitle" : "See this in action",
+                        "url" : "instruction_video.mp4" },
+                 }
+             },
+             {
+                 "identifier": "handSelection",
+                 "type": "handSelection",
+                 "title": "Which hands are you capable of doing this task with?",
+                 "inputFields":[{
+                     "uiHint": "list",
+                     "type": "singleChoice.string",
+                     "choices": [
+                         {
+                             "text": "I can only perform this activity with my LEFT hand.",
+                             "value": "left"
+                         },
+                         {
+                             "text": "I can only perform this activity with my RIGHT hand.",
+                             "value": "right"
+                         },
+                         {
+                             "text": "I can perform this activity with both hands.",
+                             "value": "both"
+                         }
+                     ]
+                 }]
+             },
+             {
+                "identifier":"eachHandInstructions",
+                "type":"instruction",
+                "title":"Each hand for 30 seconds",
+                "text":"We will do this finger-to-nose motion repeatedly for 30 seconds with one hand, and do 30 seconds with your other hand.",
+                "image":{
+                    "type":"fetchable",
+                    "imageName": "FingerToNose4",
+                    "placementType":"topBackground"
+                },
+                "actions":{
+                    "goForward":{
+                        "type":"default",
+                        "buttonTitle":"Got it"
+                    }
+                }
+             },
+             {
+                "identifier":"sitDownInstruction",
+                "type":"instruction",
+                "title":"Sit down",
+                "text":"Find a spot to sit where you can have your feet resting flat on the floor.",
+                "image":{
+                    "type":"animated",
+                    "imageNames":["FingerToNose0", "FingerToNose1"],
+                    "placementType":"topBackground",
+                    "animationDuration": 4,
```

Comment:
Switched to 4 second duration for you @syoung-smallwisdom also switched to repeat count of 1 as requested by design